test/integration/Signer.test.js

```diff
@@ -0,0 +1,124 @@
+import assert from 'assert'
```

Comment:
This test duplicates a lot of code from `StreamrClient.test.js`, especially the setup and teardown stuff. I was just wondering if maybe it would be better after all to add these test cases to `StreamrClient.test.js`?